cmake/unifex_env.cmake

```diff
@@ -18,3 +18,11 @@ elseif("x${CMAKE_CXX_COMPILER_ID}" STREQUAL "xMSVC")
 else()
   message(WARNING "[unifex warning]: unknown compiler ${CMAKE_CXX_COMPILER_ID} !")
 endif()
+
+if (MSVC)
+    # warning level 3 and all warnings as errors
```

Comment:
i gave up on enabling L4 for MSVC.  There were things it reports at this level that are just not feasible without sprinkling a lot of `#pragma`'s around and it probably isn't worth the effort.